### PR TITLE
Update the documentation README and DOCS.README

### PR DESCRIPTION
**Pull request by patrick-tolosa**
#### What does this PR solve?
This PR provides a more comprehensive 'getting started' tutorial for the front page of the JSDoc website
This PR also links from the readme page to the documentation, instead of maintaining a duplication

The syntax shown here (Cloudinary instance) will be available once this PR is approved
https://github.com/cloudinary/cloudinary-js-base/pull/196

#### Final checklist
- [ ] JSdoc - Add proper docstrings based on our PHP implementation
- [ ] JSdoc - Hide private functions using @private
- [ ] Implementation is aligned to Spec.
- [ ] Tests - Add proper tests to the added code

### CODE REVIEW

File: DOCS.README.md [hunk 1]
```diff
@@ -5,77 +5,156 @@
...
+  }
 });
 
+// Plug the image type into your instance
+cld.useImage(TransformableImage);
+
+// Lets create a new image
+const myImage = cld.image('sample');
+
+// The image interface exposes alot of useful functions we call ActionGroups
```
Comment (carlevison) [inline DOCS.README.md:44]:
```suggestion
// The image interface exposes a lot of useful functions we call ActionGroups
```

File: DOCS.README.md [hunk 2]
```diff
@@ -5,77 +5,156 @@
...
+// myImage.resize();  
+
+
+// the resize() ActionGroup needs an Action parameter
+// Lets give resize() a ResizeScale paramter to scale our image
+// We import all of our Action parameters from @cloudinary/base/actions/{actionGroup}
+// Note that you can import all Resize Actions, or just the single one you want
+import Resize, {scale} from '@cloudinary/base/actions/resize';
+// Resize.scale === scale
+// Explose the Resize object to discover more resize actions
```
[the remark this answers is not in the document]
Comment (patrick-tolosa) [reply DOCS.README.md:59]: Explore - As: Check what's inside the Resize object to discover more actions

File: DOCS.README.md [hunk 3]
```diff
@@ -5,77 +5,156 @@
...
-myImage.setConfig({
+// Create your instance
+const cld = new Cloudinary({
   cloud: {
     cloudName: 'demo'
-    }
+  },
+  url: {
+    secure: true // force http or https
+  }
```
[the remark this answers is not in the document]
Comment (patrick-tolosa) [reply DOCS.README.md:35]: Yes, it's still not by default, and I also wanted to have something in the example that has a URL configuration

File: DOCS.README.md [hunk 4]
```diff
@@ -2,80 +2,158 @@
...
-
-// Configure your image
-myImage.config(/* ...snip, your config, same as above */);
+const myImage = cld.image('sample');
 
-// the mode action accepts a rotationMode value - verticalFlip()
+// rotate() accepts a RotationModeAction (mode)
+// mode() has a Qualifier, however because that Qualifier is mandatory, we pass it directly to the factory
+// We pass it verticalFlip() as a QualifierValue to flip the image vertically
+// As you can see, QualifierValues are also functions
```
Comment (carlevison) [inline DOCS.README.md:137]:
```suggestion
// As you can see, QualifierValues are also functions:
```

**Pull request merged**